### ISSUE TITLE
test_shv_09 failure with version 2.4.0 on Debian sid

### ISSUE THREAD

**Issue by emollier** [klebgenomics/Kleborate#73]
Seasonal Greetings,

For information, when trying to package the newer kleborate 2.4.0 for Debian, I hit the following test failure:
```
FAIL: test_shv_09 (test.test_shv.TestShvMutations.test_shv_09)
This test has SHV-1 with a nonsynonymous mutation in the omega loop (so it is reported).
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/<<PKGBUILDDIR>>/.pybuild/cpython3_3.11/build/test/test_shv.py", line 118, in test_shv_09
    self.assertEqual(results['Bla_ESBL_acquired'], 'SHV-1* +174R')
AssertionError: '-' != 'SHV-1* +174R'
- -
+ SHV-1* +174R
```

Notably relevant software versions associated to the environment are:
  - python 3.11.7
  - kaptive 2.0.8 (this is also reproducible with kaptive 2.0.4)
  - biopython 1.81 (but I have not checked with 1.82 yet)

Running the older kleborate 2.3.2 test suite in the same environment goes through without any failure.  Since test_shv_09 has not changed from 2.3.2 to 2.4.0, this looks to be a regression.

Wishing you nevertheless a happy year,  :)
Étienne.

**Comment by katholt** [comment]
oops, sorry, this is because we modified SHV functionality in 2.4.0 but did not update the corresponding unit test.
this has been fixed in release 2.4.1

**Comment by emollier** [comment]
Thank you for the fix, I'm currently integrating Kleborate 2.4.1 in Debian sid, and it all good.
Have a nice day,  :)
Étienne.
